ui/app/AppLayouts/Onboarding/UseRecoveryPhraseFlow.qml

```diff
@@ -51,17 +49,21 @@ OnboardingStackView {
                 return ""
             }
 
-            onSeedphraseUpdated: function(valid, seedphrase) {
+            onSeedphraseProvided: seedPhrase => {
```

Comment:
Would it make sense to have this validation inside the component?? Why the instantiator has the ability to change that?? :thinking: